Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -181,7 +181,6 @@ void CommonAsmRoutines::GenMfcr()
   X64Reg cr_val = RSCRATCH_EXTRA;
   XOR(32, R(dst), R(dst));
   // we only need to zero the high bits of tmp once
```

Comment:
I'm using `tmp` later on in the loop for address calculations, hence the need to clear it. I'll correct the comment.